backend/workflow_manager/workflow/views.py

```diff
@@ -86,18 +86,12 @@ def _generate_workflow(self, workflow_id: str) -> WorkflowGenerator:
         return generator
 
     def perform_update(self, serializer: WorkflowSerializer) -> Workflow:
-        """To edit a workflow. Regenerates the tool instances for a new prompt.
+        """To edit a workflow.
 
         Raises: WorkflowGenerationError
         """
         kwargs = {}
-        if serializer.validated_data.get(WorkflowKey.PROMPT_TEXT):
```

Comment:
NIT: The BE code for this can be removed too - if not I suggest marking it with a TODO to not lose track

Reply:
Adddressed